cap/modules/schemas/views.py

```diff
@@ -115,6 +118,63 @@ def permissions(name=None, version=None):
         return jsonify({}), 204
 
 
+@blueprint.route(
+    '/<string:name>/notifications', methods=['GET', 'PATCH']
+)
+@blueprint.route(
+    '/<string:name>/<string:version>/notifications',
+    methods=['GET', 'PATCH'],
+)
+@get_schema
+@schema_admin_permission
+@super_admin_permission.require(http_exception=403)
+def notifications_config(name=None, version=None, schema=None, *args, **kwargs):
+    """CRUD operations for schema configuration."""
+    serialized_config = schema.config_serialize()
+    notifications_config = serialized_config.get("config", {}).get("notifications", {})
+    if request.method == "PATCH":
+        try:
+            data = request.get_json()
+            data = amend_patch_operation(data, prefix_path='/config/notifications')
```

Comment:
why do we need to amend the `patch` payload to update the `path` and we just dont do the patch in the specific value

so instead of adding `'/config/notifications'` to all paths, just get the value of `schema.config.notifications` and apply patch to this?